lib/cppcheck.cpp

```diff
@@ -420,7 +420,9 @@ unsigned int CppCheck::check(const ImportProject::FileSettings &fs)
         temp.check(Path::simplifyPath(fs.filename));
     }
     std::ifstream fin(fs.filename);
-    return temp.checkFile(Path::simplifyPath(fs.filename), fs.cfg, fin);
+    auto returnValue = temp.checkFile(Path::simplifyPath(fs.filename), fs.cfg, fin);
```

Comment:
my feeling is that `auto` can be used in these cases:
 * when type is dynamic
 * when type is very complex

Right here it's just an integer so I'd prefer we write the concrete type info.